clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -3833,6 +3833,11 @@ def note_cannot_use_trivial_abi_reason : Note<
   "it is polymorphic|"
   "it has a base of a non-trivial class type|it has a virtual base|"
   "it has a __weak field|it has a field of a non-trivial class type}1">;
+def warn_ppc_musttail_maybe_ignored: Warning<
+  "'musttail' attribute may be ignored on ppc targets">,
+  InGroup<IgnoredAttributes>;
```

Comment:
musttail can't be ignored with just a warning: This is a promise by the implementation, and this is not just an optimization, it is often required to get correct behavior on code that relies on it.

I expect this will lead to crashes due to stack exhaustion in such programs, or other UB with assembly interop.

Reply:
Thank you for giving a look. Will update.